src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private Random random;
```

Comment:
```suggestion
    private Random random = new Random();
```

Reply:
[random obj creating shifted out of meth](https://github.com/Maksym-Kulakov/jv-lottery/commit/67f43c3ecb6e72e75e2ad6b843fad1f5233acda6)